Resources/Prototypes/_LostParadise/Entities/Objects/Misc/stamps.yml

```
@@ -0,0 +1,253 @@
+#Clerance Levels
+- type: entity
+  name: rubber stamp "Confidentional"
+  parent: RubberStampBase
+  id: LPPRubberStampConfidentional
+  suffix: DO NOT MAP
+  components:
+  - type: Stamp
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)